components/profile/AccountSettings.tsx

```diff
@@ -155,12 +180,15 @@ const SettingsForm = () => {
             <Form
               layout="horizontal"
               onFinish={submitForm}
-              initialValues={{Username:currentUser.username,Email:currentUser.email,Github:currentUser.githubLink,Twitter:currentUser.twitterLink,LinkedIn:currentUser.linkedinLink,Website:currentUser.website}}
+              initialValues={{Username:currentUser.username,Email:currentUser.email,Github:currentUser.githubLink,Twitter:currentUser.twitterLink,LinkedIn:currentUser.linkedinLink,Website:currentUser.website,Name:currentUser.name,Location:currentUser.location,Occupation:currentUser.occupation}}
               form={form}
             >
               <Form.Item label="User Profile">
                 <Avatar src={userInfo.image} size={40} />
-                <label style={{ color: "black",marginLeft:"1em" }}>Reupload Image<input style={{ display: "none" }} type="file" onChange={Reupload} /></label>
+                <Styledlabel style={{ marginLeft:"1em" }}>Reupload Image<input style={{ display: "none" }} type="file" onChange={Reupload} /></Styledlabel>
+              </Form.Item>
+              <Form.Item label="Name" name="Name">
```

Comment:
Make each field its own component and pass the onChange, value and placeholder as props since this is repeated so much. Do this for all the fields that require typing in except for the bio textbox.